src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -58,7 +58,7 @@ protected Invitation()
             EndTimeUtc = endTimeUtc;
             Location = location;
             ObjectGuid = Guid.NewGuid();
-            AddDomainEvent(new IpoCreatedEvent(plant, ObjectGuid));
+            AddPreSaveDomainEvent(new IpoCreatedEvent(plant, ObjectGuid));
```

Comment:
Events dispatched before save should be renamed to something like IpoToBeCreatedEvent or something. It is not created yet...